pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
```

Comment:
Not sure exactly why I added it months ago 😅 and since we already provide the path of tests files in travis config, it doesn't need to configured on separate file.